gpu-aware-scheduling/docs/usage.md

```diff
@@ -0,0 +1,115 @@
+# Usage with NFD and GPU-plugin
+This document explains how to get GAS working together with NFD and the GPU-plugin.
+
+To begin with, it will help a lot if you have been successfully already using the GPU-plugin with some deployments. That means your HW and cluster is most likely fine with GAS also.
+
+## GPU-plugin
+Resource management enabled version of the GPU-plugin is currently necessary for running GAS. The resource management enabled GPU-plugin version can read the necessary annotations of the PODs, and without those GPU allocations will not work correctly. It can be deployed using the device plugins operator by enabling the related CRD element, or directly via yamls. Of noteworthy is the access to the pod resource grpc server.
+
+You will need this for the plugin:
+````
+        - name: podresources
+          mountPath: /var/lib/kubelet/pod-resources
+````
+
+And therefore volumes need this:
+````
+      - name: podresources
+        hostPath:
+          path: /var/lib/kubelet/pod-resources
+````
+
+You will want in your yaml also the GPU initcontainer which installs the NFD hook. In that you will need this:
+````
+        volumeMounts:
+        - mountPath: /etc/kubernetes/node-feature-discovery/source.d/
+          name: nfd-source-hooks
+````
+
+And volumes need this:
+````
+      - name: nfd-source-hooks
+        hostPath:
+          path: /etc/kubernetes/node-feature-discovery/source.d/
+          type: DirectoryOrCreate
+````
+
+Other than those, the deployment of the GPU plugin is fairly same as without resource managing enabled. You should use a high shared-dev-number, and you need to enable the resource manager:
+````
+command: ["/usr/local/bin/intel_gpu_device_plugin", "-shared-dev-num", "300", "-v", "2", "-resource-manager", "true"]
```

Comment:
Can you add a version of the full yaml for an enabled GPU Plugin here - either in text or as part of the repo?

Reply:
I need to think about that. I could write something based on our validation yamls. The plugin should work via operator at least. Usage of plugin through the kustomize thingy they have for it is another thing.

Reply:
I will point to the gpu plugin, there will be a kustomize overlay for this